README.md

```diff
@@ -1,23 +1,90 @@
+will update the versions as they become available and append the new files to the mirror.  A limited number of older versions will be maintained
+in the mirror but will be removed after a period of time.  New providers or tools can be added to the project by submitting a PR updating any of
+previously mentioned files.  New releases for this project will track the Terraform core version.  As new versions of Terraform are incorporated, a corresponding release of this project will be created.
+
+# How to use project or mirror
+
+The providers mirror created by this project can be used directly, replicated to a disconnected network, or the project itself can be used to create a Terraform network mirror.
+
+## Directly use public mirror
+
+The [`Terraform providers mirror`][5] can be used directly by creating a [`Terraform CLI Configuration file`][6] and using the following
+`provider_installation` block:
+
+```
+provider_installation {
+  network_mirror {
+    url = "https://hashicorp-mirror.cloudarmor.io/providers/"
+  }
+}
+```
+## Disconnected environments
+
+The mirror can be copied as-is onto a local system. The `provider_installation` block in the Terraform CLI Configuration file needs to be updated
+to use `filesystem_mirror` with the path to where the mirror was copied.
+
+Another option is to create an internal static website using the mirror, use the `network_mirror`` option with a url to the static website.
+
+Please see [`Provider Installation`][8] for additional information.
+
+## Create a mirror
+
+To use this project to create a mirror, simply clone this project and run `terraform providers mirror` with the desired `-platform` parameter 
+and destination path.
+
+For example,
+
+```
+terraform providers mirror -platform=linux_amd64 ./mirror/
+```
+
+To add additional providers, provider versions, or platforms, simply update `provider_versions.tf` and re-run the above command with the desired
+platform using the same mirror path.  Using the same mirror path will allow Terraform to properly update the mirror json files to properly track multple provider versions.
+
+
+[0]: https://hashicorp-mirror.cloudarmor.io/list.html
+[1]: https://developer.hashicorp.com/terraform/internals/provider-network-mirror-protocol
+[2]: https://github.com/plus3it/tardigrade-terraform-network-mirror/blob/tf-mirror/Dockerfile.tools
+[3]: https://github.com/plus3it/tardigrade-terraform-network-mirror/blob/tf-mirror/.github/workflows/dependabot_hack.yml
+[4]: https://github.com/plus3it/tardigrade-terraform-network-mirror/blob/tf-mirror/provider_versions.tf
+[5]: https://hashicorp-mirror.cloudarmor.io/providers/
+[6]: https://developer.hashicorp.com/terraform/cli/config/config-file
+[8]: https://developer.hashicorp.com/terraform/cli/config/config-file#provider-installation
+[9]: https://developer.hashicorp.com/terraform/cli/config/config-file#filesystem_mirror
+
+<!-- BEGIN TFDOCS -->
```

Comment:
Umm. Hmm. Can we just remove this section? The linter is going to fail this when dependabot updates the versions...

Reply:
Lol.  I think I added it because I got a linting error.  Is there a way to ignore the terraform-docs lint?

Reply:
Hmm. I think the setup is such that, if there is a README.md file AND IF there are any .tf files, then it will append this section to the readme. I'm not sure on a way to override that directly. But, you could move the .tf file to a subdirectory _without_ a  README file, and I bet that would work...